src/gen/module.rs

```diff
@@ -123,11 +123,11 @@ impl Module {
 }
 
 impl internal::InnerAndOuterAnnotations for Module {
-    fn inner_annotations(&mut self) -> &mut Vec<String> {
+    fn inner_annotations(&mut self) -> &mut Vec<Annotation> {
         &mut self.inner_annotations
     }
 
-    fn outer_annotations(&mut self) -> &mut Vec<String> {
+    fn outer_annotations(&mut self) -> &mut Vec<Annotation> {
         &mut self.outer_annotations
     }
 }
```

Comment:
With this change we do not need to have inner and outer anymore. Inner can only be scope Annotations and outer just item Annotations.